### PR TITLE
[ALFREDOPS-839] Add support for Alfresco 7.3

### PR DESCRIPTION
**Pull request by RVanhuysseXenit**
Also drops support for 5x.

### CODE REVIEW

File: c4a-impl/build.gradle [hunk 1]
```diff
@@ -149,6 +151,19 @@ allprojects {
         version = project.version.replace("SNAPSHOT", buildNumber)
         from(compileTypeScript.outputs)
 
+        def importPackageValue = 'org.json;version="[0,99999999)",'
```
Comment (WimCrols) [inline c4a-impl/build.gradle]: `version="0"` means [0, unbounded) so I would use that.

File: c4a-test/build.gradle [hunk 2]
```diff
@@ -116,8 +111,7 @@ subprojects {
     }
 
     dockerAlfresco {
-//        baseImage = "docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
-        baseImage = "hub.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
+        baseImage = "private.docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
```
Comment (WimCrols) [inline c4a-test/build.gradle]: Does this work? Apix uses `baseImage = 'docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:7.3.0.1'`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # Changelog
 
-## Release 2.5.1 (Unreleased)
+## Release 3.0.0 (Unreleased)
+This release drops support for Alfresco 5.x.
```
Comment (WimCrols) [inline CHANGELOG.md:4]: DE and Alfred API also drop support for 6.0 and 6.1. Up to you if you want to follow suit.

**Review by WimCrols — CHANGES_REQUESTED** [review]

File: c4a-test/build.gradle (hunk 2 above)
Comment (RVanhuysseXenit) [reply c4a-test/build.gradle]: yeah, that's new. cloudsmith implemented that on our request. Works around an issue where docker can only have one login per repo.
Now all the old namespaces count as separate repos.
Also avoids an issue where a login to an authenticated repo blocks access to an open repo. (See Thijs for more info)

File: CHANGELOG.md (hunk 3 above)
Comment (RVanhuysseXenit) [reply CHANGELOG.md:4]: 6.0 and 6.1 already were no longer supported. 6x compiles for 6.2.2 and we give no guarantees for lower versions

File: c4a-test/build.gradle (hunk 2 above)
Comment (RVanhuysseXenit) [reply c4a-test/build.gradle]: though the private element in the querypath does needs to be dropped

Comment (WimCrols) [reply c4a-test/build.gradle]: Aha! Good to know. They should @channel this kind of stuff :/

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -34,6 +34,13 @@ jobs:
           registry: private.docker.xenit.eu
           username: ${{ secrets.CLOUDSMITH_USER }}
           password: ${{ secrets.CLOUDSMITH_APIKEY }}
+      # REquired as long as jodconverter is required and not moved to SW
```
Comment (WimCrols) [inline .github/workflows/ci.yml]: What is SW?

Comment (WimCrols) [reply .github/workflows/ci.yml]: I'm thinking maybe Scaleway? I think you mean Cloudsmith.
Can you change that before committing? I'll already approve.

**Review by WimCrols — DISMISSED** [review]

**Pull request merged**